requirements_base.txt

```diff
@@ -4,7 +4,6 @@ h5py==2.6.0
 jupyter==1.0.0
 numpy==1.11.0
 Pillow==3.2.0
-pyosmium==2.6.0
```

Comment:
It gets installed from source in the Dockerfile, I think pyosmium might not be pippable at all?